src/ingest-pipeline/schemata/export_and_backup_map_schema.yml

```diff
@@ -0,0 +1,25 @@
+'$schema': 'http://json-schema.org/schema#'
+'$id': 'http://schemata.hubmapconsortium.org/export_and_backup.yml'
+'title': 'export_and_backup schema'
+'description': 'export_and_backup schema'
+
+'allOf': [{'$ref': '#/definitions/export_and_backup_map'}]
+
+'definitions':
+
+  'export_and_backup_record':
+     'type': 'object'
+     'properties':
+        'status':
+           'type': 'string'
+           'description': 'the status of the dataset or upload'
+        'plugins':
+            'type': 'array'
+            'description': 'array of appropriate plugins for status'
+  'export_and_backup_entity_type':
+      'type': 'string'
+      'description': 'contains associated export_and_backup_records'
+  'export_and_backup_map':
+    'type': 'object'
+    'properties':
+      'export_and_backup_entity_type': {'$ref': '#/definitions/export_and_backup_entity_type'}
```

Comment:
This bit doesn't seem right; it's too self-referential.  I think you need something like:
```
'type': 'object'
'properties':
  'dataset':
    'type': 'array'
    'items': {'$ref': '#/definitions/export_and_backup_record'}
    'description': 'plugins applicable to datasets'
  'upload':
    'type': 'array'
    'items': {'$ref': '#/definitions/export_and_backup_record'}
    'description': 'plugins applicable to uploads'

```

Reply:
Schemata are really tricky.  Have I told you how to test them using:
```
cd src/ingest-pipeline/misc/tools
./schema_tester.py export_and_backup_map_schema.yml ../../airflow/dags/export_and_backup_map.yml
```
Yours passes that test at the moment, but I think only because it's under-specified.

Reply:
I took a crack at this update. It still passes but I am uncertain whether to trust that or not now :)

Reply:
The schemata look right to me now.  If they pass the schema tester, I think you've got this part straightened out.